frontend/src/features/book/BookListSection.tsx

```diff
@@ -32,15 +42,23 @@ const BookListSection = ({
         setSelected={setActiveTab}
       />
       <div css={ContentContainer}>
-        {bookDataList.map((bookData, idx) => (
-          <div key={bookData.name}>
-            <BookCard
-              bookType={TabMatcher.matchBookTypeKRToENG(activeTab)}
-              data={bookData}
-            />
-            {idx !== bookDataList.length - 1 && <div css={LineWrapper} />}
-          </div>
-        ))}
+        {data && data?.length !== 0 ? (
```

Comment:
정말정말 사소하지만, length를 비교한다면 !== 0 보다는 > 0이 읽기 조금 더 편할 수 있을 것 같아요 !
```suggestion
        {data && data?.length > 0 ? (
```